Apps/WebClient/src/ClientApp/webpack.config.js

```diff
@@ -110,7 +110,7 @@ module.exports = (env, argv) => {
             ]
           : [
               // Plugins that apply in production builds only
-              new MiniCssExtractPlugin({ filename: "site.css" }),
+              new MiniCssExtractPlugin({ filename: "[hash].site.css" }),

```

Comment:
i am assuming this creates a unique hash to force the css to be pushed down? Good trick!

Reply:
Correct! What was confusing is that ASP Net uses its own version that gets chained with the chunks of each css. So in the end orifices several site.css files.